go/chat/signencrypt/codec.go

```diff
@@ -171,17 +171,15 @@ func getPacketLen(plaintextChunkLen int) int {
 	return plaintextChunkLen + secretbox.Overhead + ed25519.SignatureSize
 }
 
-func sealPacket(plaintext []byte, chunkNum uint64, encKey SecretboxKey, signKey SignKey, nonce Nonce) []byte {
-	chunkNonce := makeChunkNonce(nonce, chunkNum)
-	signatureInput := makeSignatureInput(plaintext, encKey, chunkNonce)
+func sealPacket(plaintext []byte, encKey SecretboxKey, signKey SignKey, signaturePrefix libkb.SignaturePrefix, chunkNonce SecretboxNonce) []byte {
```

Comment:
If we're making `sealPacket` more agnostic about whether it's used as a chunk in a stream or as an atomic message, we might rename the `chunkNonce` param to just `nonce`?